src/pages/home/home.css

```diff
@@ -0,0 +1,27 @@
+.Item {
```

Comment:
this and the style below are used in item-template.js not home.js so should be in their own file